src/main/java/com/autotune/experimentManager/utils/EMConstants.java

```diff
@@ -29,6 +29,11 @@
         public static String NAMESPACE = "default";
     }
 
+    public static class EMEnv {
+        public static final String AUTOTUNE_MODE = "AUTOTUNE_MODE";
+        public static final String EM_ONLY_MODE = "EM_ONLY";
+    }
+
```

Comment:
These constants should be in the `autotune/utils/AutotuneConstants.java`

Reply:
Thanks for the review, Moved constants to `autotune/utils/AutotuneConstants.java`.